src/Commands/DefaultShooter.cpp

```diff
@@ -9,43 +9,55 @@
 // it from being updated in the future.
 
 
-#include "IntakeUp.h"
+#include "DefaultShooter.h"
 
 // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
 
-IntakeUp::IntakeUp(): Command() {
+DefaultShooter::DefaultShooter(): Command() {
         // Use requires() here to declare subsystem dependencies
     // eg. requires(chassis);
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
-	Requires(Robot::intake.get());
+	Requires(Robot::shooter.get());
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
 }
 
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
 
 // Called just before this Command runs the first time
-void IntakeUp::Initialize() {
+void DefaultShooter::Initialize() {
 
 }
 
 // Called repeatedly when this Command is scheduled to run
-void IntakeUp::Execute() {
-	RobotMap::intakeintakeMotor3->Set(0.5);
-	RobotMap::intakeintakeMotor4->Set(0.5);
+void DefaultShooter::Execute() {
+	RobotMap::shootershooterMotor1->Set(0.0);
+	RobotMap::shootershooterMotor2->Set(0.0);
+	RobotMap::shooterrollerMotor->Set(0.0);
+
+	if (RobotMap::shooterliftShooter->Get() == DoubleSolenoid::kForward)
```

Comment:
We should not have this switching logic in this class. We should decide which way is up and which is down. Always set them values correctly at startup and always set them correctly in the code. For example, I expect that this one should be down.